File: DSA PRACTICE/LECTURE8/Calculator.java
// SwitchProblem:02 Calculator
import java.util.Scanner;

public class Calculator 
{
	public static void main(String[] args) 
	{
			Scanner obj=new Scanner(System.in);
			System.out.println("Enter first number");
			int a=obj.nextInt();
			System.out.println("Enter second number");
			int b=obj.nextInt();

			System.out.println("Enter the operation");
			String ch=obj.next();

			switch(ch)
		{
			case "+": System.out.print("a+b: "+(a+b));
						break;
			case "-": System.out.print("a-b: "+(a-b));
						break;
			case "/" : System.out.print("a/b: "+(a/b));
						break;
			case "*" : System.out.print("a*b: "+(a*b));
						break;
			case "%" : System.out.print("a%b: "+(a%b));
						break;
			default: System.out.print("Invalid operation");	
		}

	}
}



	
  

File: DSA PRACTICE/LECTURE8/Calculator.cpp
// SwitchProblem:02 Calculator
#include<iostream>
using namespace std;

int main()
{
	int a,b;
	cout<<"Enter first number"<<endl;
	cin>>a;
	cout<<"Enter second number"<<endl;
	cin>>b;

	char ch;
	cout<<"Enter the operation"<<endl;
	cin>>ch;

	switch(ch)
	{
		case '+': cout<<"a+b: "<<a+b;
					break;
		case '-': cout<<"a-b: "<<a-b;
					break;
		case '/' : cout<<"a/b: "<<a/b;
					break;
		case '*' : cout<<"a*b: "<<a*b;
					break;
		case '%' : cout<<"a%b: "<<a%b;
				    break;
		default: cout<<"Invalid operation";	
	}
    return 0;
}